### ISSUE TITLE
Some questions about model training

### ISSUE THREAD

**Issue by chenliang-zhang** [yingtaoluo/Spatial-Temporal-Attention-Network-for-POI-Recommendation#12]
Hi！In train.py， you use first 100 sequences as sample to train model and get great performence. But if I use the whole dataset instead of the first 100, I can't get similar recall in the paper. Is there any settings other than initial learning rate should be changed for bigger dataset?

**Comment by yingtaoluo** [comment]
Yes. As we have different user embedding in the paper, and each user is modeled individually, it is fine to train on many small proportions of users and then average the performance. Otherwise, we need to increase the model capacity/# hyperparameters such as embedding size.

**Comment by yingtaoluo** [comment]
When comparing with other baselines, the same (proportion) setting should be used, otherwise, there is a fairness problem. When training other baselines, we find it difficult to reach similar results reported in their original papers. It is a compromise.
